documentation/modules/configuring/ref-affinity.adoc

```diff
@@ -3,59 +3,68 @@
 // assembly-scheduling.adoc
 
 [id='affinity-{context}']
-= Specifying affinity, tolerations, and topology spread constraints
+= Scheduling strategies
 
-Use affinity, tolerations and topology spread constraints to schedule the pods of kafka resources onto nodes.
-Affinity, tolerations and topology spread constraints are configured using the `affinity`, `tolerations`, and `topologySpreadConstraints` properties in following resources:
+Kafka components can be scheduled onto Kubernetes nodes using affinity rules, tolerations, and topology constraints.  
+These strategies help isolate workloads, optimize resource usage, and improve overall cluster performance.
 
-* `Kafka.spec.kafka.template.pod`
-* `Kafka.spec.entityOperator.template.pod`
-* `KafkaConnect.spec.template.pod`
-* `KafkaBridge.spec.template.pod`
-* `KafkaMirrorMaker2.spec.template.pod`
+The following scheduling techniques support different deployment goals:
 
-The format of the `affinity`, `tolerations`, and `topologySpreadConstraints` properties follows the Kubernetes specification.
-The affinity configuration can include different types of affinity:
+Use pod anti-affinity to avoid critical applications sharing nodes::
+Use pod anti-affinity to prevent critical applications from being scheduled on the same disk.  
+In Kafka deployments, configure pod anti-affinity to ensure that Kafka brokers do not share nodes with other workloads, such as databases.
 
-* Pod affinity and anti-affinity
-* Node affinity
+Use node affinity to schedule workloads onto specific nodes::
+Kubernetes clusters often include nodes optimized for different workloads, such as CPU, memory, storage, or network.  
+Node affinity enables scheduling Kafka components onto nodes that match specific labels, such as `beta.kubernetes.io/instance-type` or custom labels, to optimize performance and cost.
```

Comment:
The `beta.kubernetes.io/instance-type` seems to be deprecated since Kubernetes 1.17, it's replaced by `node.kubernetes.io/instance-type`.